services/QuillLMS/client/app/bundles/Teacher/components/dashboard/welcome_modal.tsx

```diff
@@ -16,13 +22,22 @@ const WelcomeModal = ({ close, size }) => {
     />
     <div className="welcome-modal quill-modal">
       <div className="modal-body">
-        <h1>Welcome to Quill!</h1>
-        <div className="welcome-modal-text">
-          <p>Our mission as a non-profit is to help students become strong writers, so all our content is completely free to use with an unlimited number of students.</p>
+        <div className="welcome-modal-image-box">
+          <h1>Welcome to Quill!</h1>
+          <img alt="Teacher at projector in classroom" src="https://assets.quill.org/images/pages/dashboard/illustrations-classroom-activities-2.svg" />
+        </div>
+        <div>
+          <div className="welcome-modal-text">
+            <p className="welcome-modal-header">Want to see Quill&apos;s full potential?</p>
+            <p>Play around with a demo account to see sample student data and reports.</p>
+          </div>
+          <br />
+          <div className="welcome-modal-buttons">
+            <button className="quill-button contained secondary medium outlined welcome-modal-demo" onClick={goToDemo} type="button">Explore demo</button>
```

Comment:
You can just make this an anchor tag with the `quill-button` styling and then you can use a normal href rather than an onClick function for it.